bucket/ddev.json

```diff
@@ -3,6 +3,13 @@
     "description": "An open source tool that makes it simple to get local PHP development environments up and running in minutes",
     "homepage": "https://ddev.readthedocs.io/en/latest/",
     "license": "Apache-2.0",
+    "notes": [
+        "If you need to use HTTPS, you must install a custom root certificate.",
+        "Please run the following Command: ",
+        "",
+        "> mkcert -install",
+        ""
```

Comment:
Hi can you have a look at https://github.com/ScoopInstaller/Extras/blob/master/bucket/wireshark.json and update the note field to match its style?